index.d.ts

```diff
@@ -65,13 +65,14 @@ declare module 'plaid' {
   interface Credential {
     name: string;
     label: string;
-    type: 'text' | 'password';
+    type: "text" | "password";
   }
 
   interface Institution {
     credentials: Array<Credential>;
     has_mfa: boolean;
     institution_id: string;
+    logo?: string | null;
```

Comment:
makes sense. it is optional because it is only provided when `include_display_data` is true. it is nullable because not all institutions have logos right now (although most of them do).  

is that your thinking @dmurawsky?

Reply:
Yeah, exactly @ntindall